src/rsa/key_generation.rs

```diff
@@ -0,0 +1,147 @@
+// Copyright 2018 Brian Smith.
```

Comment:
If you copied some of my code, please keep this copyright line. Please add your own copyright line.